cloudwatch-connector/src/main/java/com/sequenceiq/cloudbreak/cloudwatch/processor/CloudwatchRecordWorker.java

```diff
@@ -90,20 +94,26 @@ private DescribeLogStreamsResult describeLogStreams() {
         describeLogStreams.setLogGroupName(logGroup);
         describeLogStreams.setLogStreamNamePrefix(logStream);
         try {
-            return getAwsLogsClient().describeLogStreams(describeLogStreams);
+            DescribeLogStreamsResult describeLogStreamsResult = getAwsLogsClient().describeLogStreams(describeLogStreams);
```

Comment:
Any test for the new logic?

Reply:
Added unit tests in CloudwatchRecordWorkerTest